src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,55 @@
+package duke.task;
+
+import java.io.File;
+
+import java.time.format.FormatStyle;
+
+
+public class Task {
+    private static final String DATA_PATH = new File("").getAbsolutePath() + "/data/duke.txt";
+    private String title;
+    private boolean completed;
```

Comment:
I can tell from your code that the boolean tracks whether the task is completed. Perhaps you should rename it to sound more like a boolean.
```suggestion
    private boolean isCompleted;
```